core/dbt/contracts/graph/model_config.py

```diff
@@ -53,14 +53,5 @@ class UnitTestNodeConfig(NodeConfig):
 }
 
 
-# base resource types are like resource types, except nothing has mandatory
-# configs.
-BASE_RESOURCE_TYPES: Dict[NodeType, Type[BaseConfig]] = RESOURCE_TYPES.copy()
-
-
 def get_config_for(resource_type: NodeType, base=False) -> Type[BaseConfig]:
```

Comment:
Is the optional `base` parameter still necessary? It looks unused. 

And for my own understanding, are we no longer calling this method for resource types not in `RESOURCE_TYPES`?

Reply:
It's never been used. As far as I can see the result for base=True and base=False has always been identical. I removed the other instances but missed it in the signature. Now removed.